src/sentry/issues/escalating.py

```diff
@@ -258,3 +238,39 @@ def is_escalating(group: Group) -> bool:
         )
         return True
     return False
+
+
+def parse_groups_past_counts(response: Sequence[GroupsCountResponse]) -> ParsedGroupsCount:
+    """
+    Return the parsed snuba response for groups past counts to be used in generate_issue_forecast.
+    ParsedGroupCount is of the form {<group_id>: {"intervals": [str], "data": [int]}}.
+
+    `response`: Snuba response for group event counts
+    """
+    group_counts: ParsedGroupsCount = {}
+    group_ids_list = group_counts.keys()
+    for data in response:
+        group_id = data["group_id"]
+        if group_id not in group_ids_list:
+            group_counts[group_id] = {
+                "intervals": [data["hourBucket"]],
+                "data": [data["count()"]],
+            }
+        else:
+            group_counts[group_id]["intervals"].append(data["hourBucket"])
+            group_counts[group_id]["data"].append(data["count()"])
+    return group_counts
+
+
+def _issue_category_dataset(category: GroupCategory) -> Dataset:
```

Comment:
These two functions look silly now but it will make sense once the changes from #48257 arrive.